wings/1.0/installing.md

```diff
@@ -75,7 +75,9 @@ sudo systemctl enable --now docker
 ```
 
 #### Enabling Swap
-
+::: tip Recent Linux kernel
+Since the version 6.1 of the linux kernel, swap is enabled by default. If you are running a kernel version 6.1 or newer, you can skip this step. To check your kernel version, run `uname -r`.
+:::
```

Comment:
```suggestion
::: tip Recent Linux kernel
Since the version 6.1 of the linux kernel, swap is enabled by default. If you are running a kernel version 6.1 or newer, you can skip this step. To check your kernel version, run `uname -r`.
:::

```